google-cloud-tools-plugin/src/com/google/gct/idea/CloudToolsPluginInfoService.java

```diff
@@ -21,4 +21,5 @@
  * information service across google-cloud-tools-plugin and google-account-plugin
  */
 public interface CloudToolsPluginInfoService extends PluginInfoService {
+  String getClientVersion();
```

Comment:
I know it's currently only used for Cloud Debugger, but there's nothing really Cloud Debugger specific in the value of the version. Is it possible that other APIs in the future will follow suit, and will also require us to pass the client version?